network/hybrid/network_test.go

```diff
@@ -92,13 +119,18 @@ func ExampleCreatePublicIP() {
 }
 
 func ExampleCreateNetworkInterface() {
-	ctx := context.Background()
+	groupName := config.GenerateGroupName("CreateNIC")
+	config.SetGroupName(groupName) // TODO: don't use globals
+	ctx, cancel := context.WithTimeout(context.Background(), 300*time.Second)
+	defer cancel()
 	defer hybridresources.Cleanup(ctx)
+
 	_, err := hybridresources.CreateGroup(ctx)
 	if err != nil {
-		helpers.PrintAndLog(err.Error())
+		internalutil.PrintAndLog(err.Error())
```

Comment:
I'd like to get rid of `PrintAndLog`. We could use `logrus` or something similar instead.

Reply:
Doing this would help give people guidance of real libraries they could use for logging in their projects, instead of just showing them something we wrote in a cage.

Reply:
I'd like to change the examples to test cases and use `testing.T` and have started to convert some already in this PR to demonstrate. Do you think that's a good idea?